src/secure_random.cr

```diff
@@ -1,19 +1,61 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom library is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others.
+#
+# For example:
+# ```crystal
+# SecureRandom.base64 #=> LIa9s/zWzJx49m/9zDX+VQ==
+# SecureRandom.hex    #=> c8353864ff9764a39ef74983ec0d4a38
+# SecureRandom.uuid   #=> c7ee4add-207f-411a-97b7-0d22788566d6
+# ```
 module SecureRandom
+  # Generates a random base64 string.
+  #
+  # The argument `n` determines its size, which is around `n` * 4/3.
```

Comment:
Let's keep saying it says how many bytes are generated and then mention the size increase after base64 encoding.